sdk/src/main/java/com/sygic/travel/sdk/api/model/place/PlaceDetail.kt

```diff
@@ -81,4 +83,53 @@ internal class PlaceDetail {
 
 	@SerializedName("references")
 	var references: List<Reference>? = null
+
+	fun convert(): Place{
+		val place = Place()
+		
+		place.id = id
+		place.level = level
+		place.categories = categories
+		place.rating = rating
+		place.quadkey = quadkey
+		place.location= location?.convert()
+		place.bounds = bounds?.convert()
+		place.name = name
+		place.nameSuffix = nameSuffix
+		place.perex = perex
+		place.url = url
+		place.thumbnailUrl = thumbnailUrl
+		place.marker = marker
+		place.parentIds = parentIds
+		place.detail = convertDetail()
+		
+		return place
+	}
+
+	fun convertDetail(): Detail {
+		val detail = Detail()
+		
+		val convertedTags: MutableList<com.sygic.travel.sdk.model.place.Tag> = mutableListOf()
```

Comment:
We will rename all internal model classes in order to be distinguishable from their public model classes equivalents. So the package paths wouldn't be required, just imports.